src/main/java/training/Roller/Roller.java

```diff
@@ -0,0 +1,54 @@
+package training.Roller;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.Roller.RollerConstants.ROLLER_DEFAULT_VELOCITY_RPM;
+import static training.Roller.RollerConstants.ROLLER_LOG_PATH;
+import static training.Roller.RollerDirection.kBackward;
+import static training.Roller.RollerDirection.kForward;
+
+public class Roller extends GBSubsystem {
+    private Rotation2d targetVelocity;
+    private RollerDirection direction;
+    private final IRoller motor;
+
+    public Roller() {
+        this.motor = new RollerFactory().create();
+        this.targetVelocity = Rotation2d.fromRotations(ROLLER_DEFAULT_VELOCITY_RPM);
+    }
+
+    public Rotation2d getTargetVelocity() {
+        return targetVelocity;
+    }
+
+    public void setTargetVelocity(Rotation2d targetVelocity) {
+        this.targetVelocity = targetVelocity;
+    }
+
+    public void runForward() {
```

Comment:
roller is rolling my friendooo
```suggestion
    public void rollForward() {
```